app/controllers/farmers_markets_controller.rb

```diff
@@ -0,0 +1,10 @@
+class FarmersMarketsController < ApplicationController
+
+  def index
+    @farmers_market = FarmersMarket.all
+  end
+
+  def show
+    # @farmersmarkets = Farmersmarkets.find(params[:id])
```

Comment:
it's ok for this dev branch, but let's try to avoid opening pull requests with commented out code.  I think it's better not to risk have some of it slip through the cracks.